src/OpenTelemetry.Exporter.InMemory/InMemoryExporterMetricExtensions.cs

```diff
@@ -42,11 +40,7 @@ public static MeterProviderBuilder AddInMemoryExporter(this MeterProviderBuilder
                 throw new ArgumentNullException(nameof(exportedItems));
             }
 
-            var options = new InMemoryExporterOptions();
-            configure?.Invoke(options);
-
-            var exporter = new InMemoryMetricExporter(exportedItems, options);
-            return builder.AddMetricReader(new PeriodicExportingMetricReader(exporter, options.MetricExportIntervalMilliseconds));
+            return builder.AddMetricReader(new BaseExportingMetricReader(new InMemoryExporter<Metric>(exportedItems)));
```

Comment:
we lose ability to have InMemory push exporter.. But its not likely needed as tests can just use "Pull".